app/build.gradle

```diff
@@ -91,7 +91,26 @@ dependencies {
     androidTestImplementation 'com.android.support.test:rules:1.0.1'
     androidTestImplementation 'com.android.support.test.espresso:espresso-core:3.0.1'
     androidTestImplementation 'com.android.support.test.uiautomator:uiautomator-v18:2.1.3'
-    androidTestImplementation 'org.mockito:mockito-android:2.+'
+    androidTestImplementation 'org.mockito:mockito-android:2.16.0'
+}
+
+Properties props = new Properties()
```

Comment:
What is the supposed to be doing ? signingConfig ?

Reply:
before that, we couldn't do `./gradlew build` or run the tests in one command. It was working before but I removed it earlier to suppress warnings. It did remove the warnings and didn't break the tests since we are testing unit and instrumentation tests separately. It made it harder to run the full test locally and I think we'll revert to non-parallel tests as travis introduces a lot of flakiness with parallel. Most builds time out or have weird error that we never had before that. This is just to give us the option to swap back and lets us test easily.